filament/backend/include/backend/Program.h

```diff
@@ -117,6 +117,9 @@ class Program {
     Program& specializationConstants(
             utils::FixedCapacityVector<SpecializationConstant> specConstants) noexcept;
 
+    Program& pushConstants(ShaderStage stage,
+            utils::FixedCapacityVector<char const*> constants) noexcept;
```

Comment:
I think we might have to use `CString` here because this goes through the command stream and it would be easu to get dangling pointers.

Reply:
Do we need the type of each constants here?

Reply:
done